molgenis-data-mapper/src/main/resources/templates/view-single-mapping-project.ftl

```diff
@@ -246,7 +246,8 @@
                             <div>
                                 <label for="newTargetEntityLabel">Label</label>
                                 <input type="text" value="" name="label" id="newTargetEntityLabel"
-                                       class="form-control">
+                                       class="form-control"
+                                       required minlength="1"/>
```

Comment:
But the help block says "Optionally"! I think we should instead interpret "" as null.